src/routes/github/subscription/github-subscription-delete.ts

```diff
@@ -4,24 +4,28 @@ import { isUserAdminOfOrganization } from "~/src/util/github-utils";
 import { createAppClient, createUserClient } from "~/src/util/get-github-client-config";
 
 export const GithubSubscriptionDelete = async (req: Request, res: Response): Promise<void> => {
-	const { githubToken, jiraHost, gitHubAppId } = res.locals;
+	const { githubToken, jiraHost } = res.locals;
+	const gitHubAppId = res.locals.gitHubAppId || req.body.gitHubAppId;
```

Comment:
Same here, can we trust the `req.body.gitHubAppId`? Should we instead put the `uuid` in the url path, and let the middlware to verify it?

Reply:
Uuid is already part of the query string. I don't think it would be necessary to make this additional check as passing a bogus gitHubAppId would fail on lines 14/15 unless someone also happened to obtain another users jiraHost and githubToken. Unless these 3 things are valid, I don't see an issue with using the gitHubAppId in the body.

Reply:
If the `uuid` is passed in query path as `/github/:uuid/subscription/${installationId}` etc, `gitHubAppId` should be already available from the `res.locals.gitHubAppConfig`, no need to get it from body.